media/cache.py

```diff
@@ -142,7 +146,7 @@ def item(self):
         if self.id in self.lib:
             return self.lib[self.id]
         else:
```

Comment:
Everything after the return is automatically an else block.

Reply:
But it does no harm to explicitly use an else statement :)